v21.1/create-changefeed.md

```diff
@@ -68,19 +68,22 @@ For more information on the sink URL structure, see [Use Cloud Storage for Bulk
 
 #### Query parameters
 
+{% include {{ page.version.version }}/cdc/url-encoding.md %}
+
 Query parameters include:
 
-Parameter          | <div style="width:100px">Sink Type</div>     | Description
--------------------+----------------------------------------------+-------------------------------------------------------------------
-`topic_prefix`     | [Kafka](#kafka), [cloud](#cloud-storage-sink) | Type: [`STRING`](string.html) <br><br>Adds a prefix to all topic names.<br><br>For example, `CREATE CHANGEFEED FOR TABLE foo INTO 'kafka://...?topic_prefix=bar_'` would emit rows under the topic `bar_foo` instead of `foo`.
-`tls_enabled=true` | [Kafka](#kafka)                               | Type: [`BOOL`](bool.html) <br><br>If `true`, enable Transport Layer Security (TLS) on the connection to Kafka. This can be used with a `ca_cert` (see below).
-`ca_cert`          | [Kafka](#kafka)                               | Type: [`STRING`](string.html) <br><br>The base64-encoded `ca_cert` file.<br><br>Note: To encode your `ca.cert`, run `base64 -w 0 ca.cert`.
-`client_cert`      | [Kafka](#kafka)                               | Type: [`STRING`](string.html) <br><br>The base64-encoded Privacy Enhanced Mail (PEM) certificate. This is used with `client_key`.
-`client_key`       | [Kafka](#kafka)                               | Type: [`STRING`](string.html) <br><br>The base64-encoded private key for the PEM certificate. This is used with `client_cert`.
-`sasl_enabled`     | [Kafka](#kafka)                               | Type: [`BOOL`](bool.html) <br><br>If `true`, [use SASL/PLAIN to authenticate](https://docs.confluent.io/current/kafka/authentication_sasl/authentication_sasl_plain.html). This requires a `sasl_user` and `sasl_password` (see below).
-`sasl_user`        | [Kafka](#kafka)                               | Type: [`STRING`](string.html) <br><br>Your SASL username.
-`sasl_password`    | [Kafka](#kafka)                               | Type: [`STRING`](string.html) <br><br>Your SASL password.
-`file_size`        | [cloud](#cloud-storage-sink)                  | Type: [`STRING`](string.html) <br><br>The file will be flushed (i.e., written to the sink) when it exceeds the specified file size. This can be used with the [`WITH resolved` option](#options), which flushes on a specified cadence. <br><br>**Default:** `16MB`
+Parameter          | <div style="width:100px">Sink Type</div>      | <div style="width:75px">Type</div>  | Description
```

Comment:
(tl;dr: below are "considerations" but no change is required in this specific PR)

Minor things but worth mentioning:

- these tables appear to be exactly the same across pages; it would be nice if this table was an include that could be reviewed once
- the non-content change to make "type" its own column means the diff is really big so I need to look at the whole table, but AFAICT we are just adding one line
- if these `div style` are not 1000% necessary I don't think we should be hacking the HTML, we should file issues on the docs site redesign instead -- I don't think us doing content styling inline is sustainable